app/Controllers/ScBookingController.cs

```diff
@@ -85,12 +85,12 @@ public async Task<IActionResult> CaseSelectedAsync(ScCaseSearchViewModel model)
 
             await _scBookingService.SaveScBookingInfoAsync(model);
 
-            return RedirectToAction("ConferenceType");
+            return RedirectToAction("HearingType");
         }
 
         [HttpGet]
-        [Route("~/booking/sc/conference-type")]
-        public IActionResult ConferenceType()
+        [Route("~/booking/sc/hearing-type")]
+        public IActionResult HearingType()
```

Comment:
I noticed in Lorne's API documentation that he named his new method GetAvailableBookingTypes (not hearing types) which leads me to believe that the title in the wireframes might actually be incorrect.  
@LL-scj-gov should we call these BookingTypes, ConferenceTypes or HearingTypes?

Reply:
Considering that the process is now including Conferences and Trials and will include Chambers in the future, we should go with the language that the court uses on the website.

I have confirmed that they prefer booking as this is what the process the user's would be following.

Reply:
Thanks @LL-scj-gov 
@duncan-oxd sorry I didn't pick up on this when you asked me yesterday.  I think this is a bit more complicated than a straight renaming because a bookingType can be one of 3 things
- hearing
- trial
- chambers
If it's a hearing then it will still have a hearingTypeId that needs to be passed into the SCSS API method BookingHearingAsync.  If it's a trial then we call the SCSS API method BookTrialHearingAsync which does not require a hearingTypeId.  And for chambers I assume there will be a third method called something like BookChambersHearingAsync which also won't require a hearingTypeId.

Reply:
@duncan-oxd and @molund Just a note that in the updated API and documentation, the BookTrialHearing method does now have a HearingType value.  However, we are being very deliberate that there are only two possible values: trial (9001) or Unmet Demand (20538).

The purpose is that when the fair-use attempts to book an item and continues to fail on all the user's selected dates then the unmet demand will be set the final possible outcome.